db/migrate/20181127010456_add_https_to_icons.rb

```diff
@@ -0,0 +1,16 @@
+class AddHttpsToIcons < ActiveRecord::Migration[5.2]
+  def up
+    Icon.where("url LIKE ''").where("url LIKE ''").find_each do |icon|
+      icon.url = icon.url.gsub("http:", "https:")
+      icon.save
+    end
+
+    Icon.where("url like '%dreamwidth%'").where("url like 'http:%'").find_each do |icon|
```

Comment:
Can we check against `dreamwidth.org` instead of just `dreamwidth`? Probably not an issue, but just to be consistent with the Ruby code.